pxtlib/auth.ts

```diff
@@ -20,7 +20,7 @@ namespace pxt.auth {
     const DEV_BACKEND_LOCALHOST_5500 = "http://localhost:5500"; // if running in Docker container
     const DEV_BACKEND_LOCALHOST_8080 = "http://localhost:8080"; // if not running in Docker
 
-    const DEV_BACKEND = DEV_BACKEND_PROD;
```

Comment:
is this a change we want to commit, or just for testing?

Reply:
Reverted this change to point to the staging backend (what it should have been -- prod was incorrect). Identity is working again in staging, and has been updated to support skillmap sync.